src/distributions/distribution-factory.ts

```diff
@@ -3,11 +3,14 @@ import { JavaInstallerOptions } from './base-models';
 import { LocalDistribution } from './local/installer';
 import { ZuluDistribution } from './zulu/installer';
 import { AdoptDistribution, AdoptImplementation } from './adopt/installer';
+import { AdoptiumDistribution, AdoptiumImplementation } from './adoptium/installer';
 
 enum JavaDistribution {
   Adopt = 'adopt',
   AdoptHotspot = 'adopt-hotspot',
   AdoptOpenJ9 = 'adopt-openj9',
+  Adoptium = 'adoptium',
+  AdoptiumHotspot = 'adoptium-hotspot',
```

Comment:
line should be removed (there are explicitly no plans for an Eclipse Temurin opnej9 build, so no need for a "-hotspot" variant name).